src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/clients/DatabaseMessageConsumerTest.java

```diff
@@ -116,21 +107,6 @@ private CaseEventMessageEntity createCaseEventMessageEntity() {
         return caseEventMessageEntity;
     }
 
-    @Test
-    void should_not_process_message_if_launch_darkly_flag_disabled() throws IOException {
```

Comment:
Should we be removing this test when the assertion is to do a particular thing based on the feature flag? I.e. removing the flag should be sufficient and the test should execute with the expected R2 journey.

Reply:
We have removed the logic of R1 route code as part of this changes, so R1 tests (when_feature_flag_disabled) have to be removed too.